app/src/main/java/com/nextcloud/talk/utils/ApiUtils.java

```diff
@@ -191,6 +192,16 @@ public static int getChatApiVersion(UserEntity user, int[] versions) throws NoSu
         throw new NoSupportedApiException();
     }
 
+    public static int getChatApiVersion(User user, int[] versions) throws NoSupportedApiException {
```

Comment:
`@Deprecate` the `UserEntity` version if possible

Reply:
DONE 👍